src/seedu/addressbook/AddressBook.java

```diff
@@ -513,13 +513,17 @@ private static String getMessageForPersonsDisplayedSummary(ArrayList<String[]> p
 	 */
 	private static ArrayList<String[]> getPersonsWithNameContainingAnyKeyword(Collection<String> keywords) {
 		final ArrayList<String[]> matchedPersons = new ArrayList<>();
+		getAllPersons(keywords, matchedPersons);
+		return matchedPersons;
+	}
+
+	private static void getAllPersons(Collection<String> keywords, final ArrayList<String[]> matchedPersons) {
 		for (String[] person : getAllPersonsInAddressBook()) {
 			final Set<String> wordsInName = new HashSet<>(splitByWhitespace(getNameFromPerson(person)));
```

Comment:
Indentation problem here. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.